pkgs/top-level/all-packages.nix

```diff
@@ -14870,7 +14870,7 @@ with pkgs;
   lttv = callPackage ../development/tools/misc/lttv { };
 
   luaformatter = callPackage ../development/tools/luaformatter
-    (lib.optionalAttrs stdenv.isDarwin {
+    (lib.optionalAttrs (stdenv.hostPlatform.system == "x86_64-darwin") {
       stdenv = overrideCC stdenv llvmPackages_9.clang;
```

Comment:
```suggestion
      stdenv = clang9Stdenv;
```
nit

Reply:
I applied your proposal.

Reply:
I had to revert it, because of following error:
```
error: undefined variable 'clang9Stdenv'

       at /nix/store/cmfmkf0lk2xyizhfkrd52mi3ksrw4d0k-source/pkgs/top-level/all-packages.nix:14885:16:

        14884|     (lib.optionalAttrs (stdenv.hostPlatform.system == "x86_64-darwin") {
        14885|       stdenv = clang9Stdenv;
             |                ^
        14886|     });
```

Reply:
Oops, looks like that is an alias. Sorry for the noise.